docs/04-device-support.md

```diff
@@ -0,0 +1,15 @@
+
+# Supported devices
```

Comment:
This section should be updated for sense

Reply:
See comment above. Maybe it's ok for MVP? Ideally, I'd like someone with a bit more hardware/testing exp to help inform this. I've only ever used Sense with a pi3 and 4 :p

Reply:
@chrisys most likely will have an idea